Modules/Core/Common/wrapping/itkImageToImageFilterB.wrap

```diff
@@ -32,13 +32,16 @@ itk_wrap_class("itk::ImageToImageFilter" POINTER)
   itk_wrap_image_filter_combinations("${WRAP_ITK_COMPLEX_REAL}" "${WRAP_ITK_SCALAR}")
 
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    foreach(t ${WRAP_ITK_SCALAR})
-      # scalar <-> SymmetricSecondRankTensor
-      itk_wrap_template("${ITKM_I${t}${d}}${ITKM_ISSRT${ITKM_D}${d}${d}}" "${ITKT_I${t}${d}}, ${ITKT_ISSRT${ITKM_D}${d}${d}}")
-      itk_wrap_template("${ITKM_ISSRT${ITKM_D}${d}${d}}${ITKM_I${t}${d}}" "${ITKT_ISSRT${ITKM_D}${d}${d}}, ${ITKT_I${t}${d}}")
+    foreach(t2 ${WRAP_ITK_REAL})
+      foreach(t ${WRAP_ITK_SCALAR})
+        # scalar <-> SymmetricSecondRankTensor
+        itk_wrap_template("${ITKM_I${t}${d}}${ITKM_ISSRT${ITKM_${t2}}${d}${d}}" "${ITKT_I${t}${d}}, ${ITKT_ISSRT${ITKM_${t2}}${d}${d}}")
+        itk_wrap_template("${ITKM_ISSRT${ITKM_${t2}}${d}${d}}${ITKM_I${t}${d}}" "${ITKT_ISSRT${ITKM_${t2}}${d}${d}}, ${ITKT_I${t}${d}}")
+      endforeach()
+      # SymmetricSecondRankTensor -> SymmetricSecondRankTensor
+      itk_wrap_template("${ITKM_ISSRT${ITKM_${t2}}${d}${d}}${ITKM_ISSRT${ITKM_${t2}}${d}${d}}"
+                        "${ITKT_ISSRT${ITKM_${t2}}${d}${d}}, ${ITKT_ISSRT${ITKM_${t2}}${d}${d}}")
```

Comment:
We can also support the likes of SymmetricSecondRankTensor[float] -> SymmetricSecondRankTensor[double] by instead using:
```cmake
      # SymmetricSecondRankTensor -> SymmetricSecondRankTensor
      foreach(t3 ${WRAP_ITK_REAL})
        itk_wrap_template("${ITKM_ISSRT${ITKM_${t3}}${d}${d}}${ITKM_ISSRT${ITKM_${t2}}${d}${d}}"
                          "${ITKT_ISSRT${ITKM_${t3}}${d}${d}}, ${ITKT_ISSRT${ITKM_${t2}}${d}${d}}")
      endforeach()
```

Reply:
Thanks @Leengit , this is a good idea. I've added the changes, will squash commits once CI completes.